src/DateTimePicker.jsx

```diff
@@ -250,7 +251,8 @@ DateTimePicker.propTypes = {
 		React.PropTypes.string
 	]),
 	dateOnly: React.PropTypes.bool,
-	forceCalendar: React.PropTypes.bool
+	forceCalendar: React.PropTypes.bool,
+	onChangeCallback: React.PropTypes.func
```

Comment:
Our convention has been to match the prop name to standard event names; in this case, `onChageCallback` would just be `onChange`